### PR TITLE
Accommodating CPS breaking changes

### PR DESCRIPTION
**Pull request by davilimap**
Updating to latest CPS package
Updating Composition package

### CODE REVIEW

File: .nuget/packages.config [hunk 1]
```diff
@@ -2,9 +2,10 @@
 <packages>
   <package id="ILMerge" version="2.14.1208" targetFramework="net45" />
   <package id="Microsoft.Build" version="15.1.262-preview5" targetFramework="net45" />
-  <package id="Microsoft.VisualStudio.Composition" version="15.0.30-pre" targetFramework="net45" />
+  <package id="Microsoft.VisualStudio.Composition" version="15.8.98" targetFramework="net45" />
```
Comment (nkolev92) [inline .nuget/packages.config:5]: Why is this needed?

Comment (davilimap) [reply .nuget/packages.config:5]: I was getting some build errors because the latest version of CPS references this (newer) version of Composition.

**Review by jainaashish — APPROVED** [review]
Looks good

**Comment by nkolev92** [comment]
Signing failed. 
I have triggered the buitd again.

**Comment by davilimap** [comment]
I'm not 100% familiar with the build system, but it seems like there are 2 types of functional tests failing
1. A group of tests fail because it expects the previous version of .Net Core. I'm not sure where this version come from, CPS shouldn't be referencing .Net
2. Another group fails because of the size of the Nuget package downloaded. Seems like the downloaded package is larger than expected, what would that mean?

**Comment by nkolev92** [comment]
@davilimap 

The functional tests are known infrastructure problems. 
The ones with the download sizes are **actively** being broken by the ongoing repo signing on NuGet.org so we don't fix them immediately. 

The end-to-end tests/apex are different though as they are running on top of a 15.8 installation of Visual Studio. 
Using this new package might be causing false negatives.

**Comment by rrelyea** [comment]
@davilimap - I talked to @nkolev92. Sounds like we should get VS 16.0 w/ appropriate changes installed with these bits and then run end-to-end tests on it.
Likely grab nuget.tools.vsix from this builds artifacts and try to install on a VS16.0 build.
image

**Comment by rrelyea** [comment]
From @davilimap : I built the repo locally with my changes and attached them to the build produced from our breaking changes branch and it seems to be working well and not causing .net core build errors anymore. I’m not sure how to run end-to-end tests on that installation of VS though.

Us: We’ll merge into Dev, optimistically.
Once that build is done, we can create our normal insertion PR for VS…and then you can merge that change into your bigger breaking change PR.
